tests/Test/Xrefcheck/LocalSpec.hs

```diff
@@ -1,25 +0,0 @@
-{- SPDX-FileCopyrightText: 2019 Serokell <https://serokell.io>
- -
- - SPDX-License-Identifier: MPL-2.0
- -}
-
-module Test.Xrefcheck.LocalSpec where
-
-import Universum
-
-import Test.Tasty (TestTree, testGroup)
-import Test.Tasty.HUnit (testCase, (@?=))
-
-import Xrefcheck.Core (canonizeLocalRef)
-
-test_local_refs_canonizing :: TestTree
-test_local_refs_canonizing = testGroup "Local refs canonizing" $
-    [ testCase "Strips ./" $
-        canonizeLocalRef "./AnchorsSpec.hs" @?= "AnchorsSpec.hs"
-
-    , testCase "Strips ././" $
-        canonizeLocalRef "././AnchorsSpec.hs" @?= "AnchorsSpec.hs"
-
-    , testCase "Leaves plain other intact" $
-        canonizeLocalRef "../AnchorsSpec.hs" @?= "../AnchorsSpec.hs"
-    ]
```

Comment:
And it would be cool to eventually add unit tests on our new utility functions, at least on non-trivial ones, instead of these tests. I believe this will help us a lot in the future.

And unlike this module, at least part of those tests will have to be aware of the platform they are run in.

Reply:
Right! I will add tests for our `CanonicalPath` utility functions.